### PR TITLE
arrow particles and misc fixes

### PR DESCRIPTION
**Pull request by ezrn**
- #404 
- #419 
- #416 
- #427 
- Added unique and differentiable particle effects to every special arrow
- All assassin arrow effects look like "potions" for better differentiability from ranger arrows
- Misc other issues fixed

### CODE REVIEW

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/bow/MarkedForDeath.java [hunk 1]
```diff
@@ -76,7 +77,7 @@ public void onHit(Player damager, LivingEntity target, int level) {
 
     @Override
     public void displayTrail(Location location) {
-        Particle.REDSTONE.builder().location(location).color(5, 5, 5).count(3).extra(0).receivers(60, true).spawn();
+        location.getWorld().spawnParticle(Particle.SPELL_MOB, location, 1, 0.1, 0.1, 0.1, 0);
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/bow/MarkedForDeath.java:80]: Any reason you didnt use the builder here? I believe it gives better control over some things e.g. view distance to see the particle

Comment (ezrn) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/bow/MarkedForDeath.java:80]: I did not know that the particle builder supported particle displacement/other particles besides dust and I'm also not sure it works to do the cheaty little hack I did for the MOB_SPELL where the delta values turn to rgb values when the particle number is set to 0. I'll try it and see if it works with the specifications I need

Comment (ezrn) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/bow/MarkedForDeath.java]: fixed

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/bow/Overcharge.java [hunk 2]
```diff
@@ -120,6 +119,33 @@ public void onPlayerShoot(EntityShootBowEvent event) {
...
+    public void createRedDustParticles() {
+        bonus.forEach((arrow, bonusDamage) -> {
+            if (arrow.isValid() && !arrow.isDead() && !arrow.isOnGround() && bonus.get(arrow) > 0) {
+
+                double baseSize = 0.25;
+                double count = (bonus.get(arrow));
+
+                double finalSize = baseSize * count;
+
+                System.out.print((float)finalSize);
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/bow/Overcharge.java:132]: oof

**Review by Mykindos — CHANGES_REQUESTED** [review]

**Pull request merged**